dist/steps/run_tests.sh

```diff
@@ -7,6 +7,69 @@
 UNITY_PROJECT_PATH="$GITHUB_WORKSPACE/$PROJECT_PATH"
 echo "Using project path \"$UNITY_PROJECT_PATH\"."
 
+#
+# Create an empty project for testing if in package mode
+#
```

Comment:
Should this whole block be moved after all the general settings?

Right before the line that says `# Overall info`

Reply:
Makes sense; I moved it!